bindings/python/setup.py.cmake.in

```diff
@@ -1,15 +1,33 @@
 from setuptools import setup
-import platform
+
```

Comment:
if it's possible, I think it would be great to get rid of this dependency on first having built libtorrent before building the python module. this `.in` file depends on `cmake` having build libtorrent first. When building with the automake (not a thing in master, and this probably doesn't work in master) there are two files created in the `bindings/python` directory, `link_flags` and `compile_flags`, just to ensure the python module is built in a compatible way to the main library.

for the python module, I think it would be reasonable to assume the main library was built with the default options (deprecated functions enabled and linking against openssl).